src/concepts/terminology.md

```diff
@@ -1,12 +1,236 @@
+
+## pipeline
+
+Mozilla’s data pipeline, which is used to collect Telemetry data from Mozilla’s products and logs from various services.
+The bulk of the data that is handled by this pipeline is Firefox Telemetry data. The same tool-chain is used to collect, store, and analyze data that comes from many sources.
+
+The code for the ingestion pipeline is available in the [gcp-ingestion](https://github.com/mozilla/gcp-ingestion) repository.
+
+## probe
+
+Measurements for a specific aspect of Firefox are called probes. A single telemetry ping sends many different probes. Probes are either Histograms (recording distributions of data points) or Scalars (recording a single value).
+
+You can search for details about probes by using the [Probe Dictionary](https://probes.telemetry.mozilla.org/). For each probe, the probe dictionary provides:
+
+* A description of the probe
+* When a probe started being collected
+* Whether data from this probe is collected in the release channel
+
+## profile
+
+All of the changes a user makes in Firefox, like the home page, what toolbars you use, installed addons, saved passwords and your bookmarks, are all stored in a special folder, called a profile. Telemetry stores archived and pending pings in the profile directory as well as metadata like the client ID. See also [Profile Creation](https://docs.telemetry.mozilla.org/concepts/profile/profile_creation.html)
+
+## query
+
+TBD
+
+## retention
+
+* As in “Data retention” - how long data is stored before it is automatically deleted/archived
+
+* As in “User retention” - how likely is a user to continue using a product
+
+## Retention
+
+### 1-Week New Profile Retention
+
+Among new profiles created on the day specified, what proportion (out of 1) are
+active during the week beginning one week after the day specified.
+
+### 1-Week Retention
+
+Among profiles that were active at least once in the week starting on the
+specified day, what proportion (out of 1) are active during the following week.
+
+## schema
+
+A schema is the organization or structure for a database. The activity of data modeling leads to a schema. Mozilla’s source schema is available on the [mozilla-services/socorro GitHub repository](https://raw.githubusercontent.com/mozilla-services/socorro/main/socorro/schemas/crash_report.json).
+
+## session
+
+The period of time that it takes between Firefox being started until it is shut down.
+
+* **Subsession**: `Sessions` are split into `subsessions` after every 24-hour time period has passed or the environment has changed.
+([for more details](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/concepts/sessions.html?highlight=subsession))
+
+## structured ingestion
+
+Mozilla’s term for the URI scheme that is used for submitting data to the Telemetry pipeline; contrast this term with “telemetry ingestion” which represents an older URI scheme that is still supported for compatibility with desktop Firefox telemetry
+
```

Comment:
Let's skip this one, it's hard to explain and not necessary for most people to know.

```suggestion
```